js/pages/questionnaire.js

```diff
@@ -347,6 +367,23 @@ export const blockParticipant = () => {
 
 }
 
+const buildSoccerResults = async (title, task) => {
+
+    let soccerURL = '';
+
+    if(location.host === urls.prod) soccerURL = SOCcerProd;
+    else if(location.host === urls.stage) soccerURL = SOCcerStage;
+    else soccerURL = SOCcerDev;
+    
+    let soccerResults = await (await fetch(`${soccerURL}title=${title}&task=${task}&n=6`)).json();
```

Comment:
Is error handling needed in for data fetching?

Reply:
Per Daniel, no.